wp-web-push/wp-web-push.php

```diff
@@ -4,6 +4,8 @@
 Text Domain: wpwebpush
 */
 
+load_plugin_textdomain('wpwebpush', '', dirname(plugin_basename(__FILE__)) . '/lang');
```

Comment:
We're still loading it in wp-web-push-main.php, but we can avoid loading it in a follow-up (if it really matters, it probably doesn't really add much to the loading time).